ACKNOWLEDGEMENTS

```diff
@@ -30,24 +9,86 @@ constructive criticism.
 As an open source project, 2009-present:
 
   * Philippe Antoine
-  * Derek Bruening
-  * Hyoun Kyu Cho
+  * Sotiris Apostolakis (Google)
+  * Nahome Bete (Google)
+  * Derek Bruening (Google)
+  * Hongyi Chen
   * Branden Clark
+  * Brett Coon (Google)
+  * Enrico Deiana (Google)
   * Shawn M. Denbow
+  * Connal de Souza (Google)
+  * Victor Derks
+  * Matthew Devereau (Arm)
+  * Pavel Durov
+  * Yuri Ershov
+  * Doug Evans (Google)
   * Peter Feiner
-  * Peter Goodman
+  * Vasilis Flouris
+  * John F.X. Galea
+  * Jack Gallagher (Arm)
+  * Peter Goodman (Google)
+  * Ryan Goulden
+  * Al Grant (Arm)
+  * Hendrik Greving (Google)
+  * Edmund Grimley Evans (Arm)
+  * Sahil Guptah (Google)
+  * Florian Hahn (Arm)
+  * Jivan Hakobyan
+  * Assad Hashmi (Arm)
   * Byron Hawkins
-  * Timur Iskhodzhanov
+  * Yang Hu
+  * Kyle Huey
+  * Timur Iskhodzhanov (Google)
+  * Mateusz Kalinowski (Arm)
+  * Svilen Kanev (Google)
+  * Stanislaw Kardach
   * Piyus Kedia
-  * Reid Kleckner
+  * Mazen Kharbutli (Google)
+  * Dhiru Kholia
+  * Yury Khrustalev
+  * Reid Kleckner (Google)
+  * Snehasish Kumar (Google)
+  * Daniel Kuts
+  * Hyoun Kyu Cho
   * Ilya Leoshkevich
+  * Ilya Leoshkevich
```

Comment:
Duplicate name.

Reply:
Thanks, fixed.  This was constructed with uniq across names plus emails, with a step to dedup once affiliations were in there: but for 4+ commits.  I went back and added the 2 or 3 commit folks but skipped the dedup step...

Reply:
No worries! :)
That specific name popped out because the two instances were one after the other.
I also double checked, there are no other duplicates.